### PR TITLE
Update SKLL to v2.1

### LINKED ISSUE #404: Update SKLL dependency to v2.1.0
We need it esp. for #403.

### PR DESCRIPTION
**Pull request by desilinguist**
This PR closes #404 by updating the SKLL version. This required changing the test data for a few SKLL-specific tests.

### CODE REVIEW

**Comment by aloukina** [comment]
Looks good to me. All changes to evaluations look negligible.

**Comment by aloukina** [comment]
Have you actually re-trained SKLL models for RSMPredict?

**Comment by desilinguist** [comment]
Only for the test that was failing? I guess I should do all of them ... ?

**Comment by aloukina** [comment]
That would be great - it should only take a minute.

**Comment by desilinguist** [comment]
Sure, let me do that.

File: tests/update_skll_models.py [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-This script is designed to update the sci-kit learn model files to ensure they
-are compatible with the current version.
-The script goes through all tests in the data/experiments directory, finds the model files and
-saves them again.
-Note that this will overwrite the original test data.
-Before running this script, make sure that the outputs work as expected
+Update SKLL models in tests.
+
+This script updates the SKLL model files to ensure they are compatible with the
+current version. It simply through all the rsmprednt tests under the
```
Comment (aloukina) [inline tests/update_skll_models.py:7]:
```suggestion
current version. It simply iterates through all the rsmprednt tests under the
```

Comment (mulhod) [reply tests/update_skll_models.py]: Is `rsmprednt` a typo too? (Sorry, wouldn't have noticed this if not for @aloukina's suggestion.)

Comment (desilinguist) [reply tests/update_skll_models.py]: Yep, already fixed 👍

**Pull request merged**